monitoring/init_pis.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+pswd="piman"
```

Comment:
see comments above. you might want to put this is a separate script and source it to get everything set once